src/Semi.Avalonia/Controls/NotificationCard.axaml

```diff
@@ -37,7 +37,6 @@
                                     Height="16"
                                     Margin="16,16,12,0"
                                     VerticalAlignment="Top"
-                                    Data="{DynamicResource InformationIconPathData}"
```

Comment:
Replace with existing info icon path.

Reply:
> Replace with existing info icon path.

Avalonia can recognize `:information` pseudoclass and use it as the default style for `NotificationCard`.
Just in case, I've replaced it.